jsonnet/thanos-ruler.jsonnet

```diff
@@ -145,6 +145,26 @@ local authorizationRole =
         secret.mixin.metadata.withNamespace(thanosRulerConfig.namespace) +
         secret.mixin.metadata.withLabels({ 'k8s-app': 'thanos' }),
 
+      // https://thanos.io/components/rule.md/#configuration
+      alertmanagerConfigSecret: {
+        kind: 'Secret',
+        alertmanagers: [
```

Comment:
is this correct? `assets/thanos-ruler/alertmanager-config-secret.yaml` doesn't look like a valid Secret.

Reply:
I haven't tested the alertmanager part, so might remove this and do in a followup. Or if you want to do this in a follow up @pgier

Reply:
removed for now